src/Plutarch/Extra/Tagged.hs

```diff
@@ -48,7 +48,56 @@ import Plutarch.TryFrom (PTryFrom (PTryFromExcess, ptryFrom'))
 import Plutarch.Unsafe (punsafeCoerce)
 import qualified PlutusTx
 
-{- | Plutarch-level 'Tagged'.
+{- | Plutarch-level 'Tagged'. @PTagged@ allows one to "tag" a given type in the typelevel.
+ Typically, @tag@ will hold `Symbol` or custom kind.  Since @PTagged@s only act as a wrapper, it does not hold any significance in the resulting script, and there are no affect on performance/script size.
```

Comment:
"will be a 'Symbol', or some other custom kind"

"Since @PTagged@s only act as wrappers, their \'tags\' don't hold any significance in the resulting script, and have no effect on performance or script size."